### PR TITLE
Skeleton states Case 2 CHW Registration

### PR DESCRIPTION
**Pull request by gsvr**
https://trello.com/c/d7yEdlM5/3-skeleton-states-case-2-chw-registration

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review

**Comment by imsickofmaps** [comment]
@smn ready for review

**Comment by imsickofmaps** [comment]
@smn PS - this is a clone of flow 1 with one state missing ("Select month baby due") - no new logic

**Comment by gsvr** [comment]
@smn It's missing 2 states - "Enter clinic code" also removed.

File: src/chw.js [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+                even += id.charAt(i + 1);
+            }
+            even = '' + even * 2;
+            for (i = 0; even.charAt(i); i++) {
+                c = even.charAt(i);
+                sum += +c;
+            }
+            sum = 10 - ('' + sum).charAt(1);
+            return ('' + sum).slice(-1) == check;
+        };
```
Comment (smn) [inline src/chw.js:88]: Next PR please put this stuff in a common lib somewhere.

**Comment by smn** [comment]
:+1: but still no contact saving?

**Pull request merged**